site/app/templates/grading/simple/Display.twig

```diff
@@ -109,18 +98,36 @@
         </select>
     </div>
     {% if action == 'numeric' and core.getUser().accessAdmin() %}
-        <div class="column-wrapper">
-            <label for="csvUpload">Upload CSV
-                <input class="csvButtonUpload" type="file" id="csvUpload" accept=".csv, .txt">
-            </label>
-            <p>
-                The CSV file should be formatted as such:
-                <br />
-                user id,first name,last name,grade1,grade2,...,total points earned,text1,text2,...
-            </p>
-        </div>
+		<p style="margin-bottom: 1vh">
+			The CSV file should be formatted as such:
+			<br />
+			user id,first name,last name,grade1,grade2,...,total points earned,text1,text2,...
+		</p>
+		<label for="csvUpload">Upload CSV
+			<input class="csvButtonUpload" type="file" id="csvUpload" accept=".csv, .txt">
```

Comment:
Can the ID and class be converted to be of the form `csv-upload` as per the Submitty conventions [here](https://submitty.org/developer/coding_style_guide/css)?